tests/pass-dep/concurrency/linux-futex.rs

```diff
@@ -1,4 +1,5 @@
 //@only-target: linux
+//@only-target: android
```

Comment:
I haven't renamed it since I don't understand the naming rule, but I guess it should be prefixed by `libc` now, right?

Reply:
No, this is not a libc thing, it's a Linux kernel thing... we don't have clear names to distinguish them (probably should be using Linux vs GNU/Linux). So the existing name is fine for now.

Reply:
Yeah, it's a Linux thing, but the naming is a bit confusing from the code perspective. Perhaps, it can be named as `linux-like` as in `std`.

Reply:
I'm fine with keeping the name here for now.